sympy/functions/elementary/exponential.py

```diff
@@ -526,7 +526,10 @@ def eval(cls, arg, base=None):
             except ValueError:
                 pass
             if base is not S.Exp1:
-                return cls(arg)/cls(base)
+                if arg.is_symbol or base.is_symbol:
+                    return cls(arg, base, evaluate=False)
```

Comment:
This should be `return None`, which causes `eval` to return the object unevaluated automatically.